superset-frontend/src/cccs-viz/plugins/plugin-chart-at-a-glance-user-id/src/AtAGlanceUserID.tsx

```diff
@@ -79,51 +79,55 @@ function AtAGlanceUserIDCore(initialFormData: QueryFormData) {
   ) {
     const filter = initialFormData.formData.extraFormData.filters[i];
     if (filter.col === 'user_id') {
-      const localuserID: string = filter.val[0];
-      if (localuserID !== userIDString) {
+      const localuserId: string = filter.val[0];
```

Comment:
Is localuser kind of one word like localhost, or should this be localUserId?

Reply:
done